packs/azure/pack.yaml

```diff
@@ -9,6 +10,6 @@ keywords:
   - servers
   - virtual machines
   - azure virtual machines
-version : 0.1
-author : st2-dev
-email : info@stackstorm.com
+version : 0.1.0
+author : StackStorm, Inc.
+email : info@stackstorm.com
```

Comment:
Lol, yeah, just a personal preference.

I couldn't resist adding `.strip()` to the script which populated metadata files with ref field :D